shuffle/cli/src/main.rs

```diff
@@ -51,9 +62,23 @@ pub async fn main() -> Result<()> {
         } => console::handle(
             &home,
             &shared::normalized_project_path(project_path)?,
-            shared::normalized_network(&home, network)?,
-            &normalized_key_path(&home, key_path)?,
-            normalized_address(&home, address)?,
+            shared::normalized_network_url(&home, network.clone())?,
+            &normalized_key_path(
+                NetworkHome::new(
+                    home.get_networks_path()
+                        .join(normalized_network_name(network.clone()))
+                        .as_path(),
+                ),
+                key_path,
+            )?,
+            normalized_address(
+                NetworkHome::new(
+                    home.get_networks_path()
+                        .join(normalized_network_name(network))
+                        .as_path(),
```

Comment:
As discussed offline, let's place this repeated `NetworkHome` construction pattern inside a factory method in `Home`.

`home.new_network_home(normalized_network_name(network))` or some better name.